internal/sink/sink.go

```diff
@@ -719,12 +741,13 @@ func (v2 *WebhookV2) updateUrls(urlCount int, url string, urls []string) {
 
 // worker is the routine that actually takes the queued messages and delivers
 // them to the listeners outside webpa
-func (v2 *WebhookV2) send(secret, acceptType string, msg *wrp.Message) error {
+func (v2 *WebhookV2) send(msg *wrp.Message, ch chan struct{}) error { //TODO: this will change when we add in content-type logic
 	defer func() {
 		if r := recover(); nil != r {
 			// s.DropsDueToPanic.With(prometheus.Labels{metrics.UrlLabel: s.id}).Add(1.0)
 			v2.logger.Error("goroutine send() panicked", zap.String("id", v2.id), zap.Any("panic", r))
 		}
+		<-ch
```

Comment:
this ch has the potential of causing chaos and it's not aiding to the cause (even tho it may feel like it is)